src/star/star_align_reads/config.vsh.yaml

```diff
@@ -72,6 +72,12 @@ argument_groups:
         description: The output file containing the splice junctions.
         direction: output
         example: splice_junctions.tsv
+      - type: file
+        name: --transcriptome
```

Comment:
Would you mind if we called this argument `--aligned_reads_to_transcriptome` or something like this? Because it's not the transcriptome that we're returning, but rather it's the alignment of the reads to the transcriptome (as opposed to the genome). 

Also, is there an argument we need to set in order to get this output? quantMode TranscriptomeSAM? In this case, we might want to mention it.

Reply:
Yes, good point. I've updated the argument name and the description.